src/main/java/frc/robot/Constants.java

```diff
@@ -264,31 +262,30 @@ public static class ExtenderConstants {
     /** This contains constants for our Gripper Pivot. */
     public static class GripperPivotConstants {
 
-      public static final Rotation2d MinAngleBase = Rotation2d.fromDegrees(-25); // TODO tune this
-      public static final Rotation2d MinAngleLow = Rotation2d.fromDegrees(-70); // TODO tune this
-      public static final Rotation2d MinAngleStandard = Rotation2d.fromDegrees(-113); // TODO: [-140, 0] maps to the same CW+ as the base pivot. Are we okay with only negative numbers?
-      public static final Rotation2d MaxAngleStandard = Rotation2d.fromDegrees(-30);
-      public static final Rotation2d MaxAngleHigh = Rotation2d.fromDegrees(60); // TODO tune this
       public static final Angle TrueSafeAngle = Degrees.of(-25);
 
-      // Gripper pivot safety ranges
-      public static final List<GripperPivotSafety> safeties = new ArrayList<GripperPivotSafety>() {{
-          add(new GripperPivotSafety(Meters.of(-0.05), Meters.of(0.3), Degrees.of(-50), Degrees.of(-25)));
-          add(new GripperPivotSafety(Meters.of(0.3), Meters.of(0.5), Degrees.of(-115), Degrees.of(5)));
-          add(new GripperPivotSafety(Meters.of(0.5), Meters.of(1.8), Degrees.of(-115), Degrees.of(60)));
+      public static final GripperPivotSafety low = new GripperPivotSafety(Meters.of(-0.05), Meters.of(0.2), Degrees.of(-50), Degrees.of(-25));
+      public static final GripperPivotSafety mid = new GripperPivotSafety(Meters.of(0.2), Meters.of(0.5), Degrees.of(-115), Degrees.of(5));
+      public static final GripperPivotSafety high = new GripperPivotSafety(Meters.of(0.5), Meters.of(1.8), Degrees.of(-115), Degrees.of(60));
+     
+      public static final List<GripperPivotSafety> Safeties = new ArrayList<GripperPivotSafety>() {{
+          add(low);
+          add(mid);
+          add(high);
         }};
 
       public static final double kP = 30.0;
       public static final double kI = 0.0;
       public static final double kD = 0.0;
-      public static final double kG = 0.43;
+      public static final double kG = 0.0; // Don't use this
       public static final double kS = 0.25;
       public static final double kV = 0.0;
       public static final double kA = 0.00;
+      public static final double dynamicKg = 0.43;
```

Comment:
We now generate our own kG on the fly - this is the max kG when the gripper is parallel to the ground